etc/deploy/aws.sh

```diff
@@ -235,6 +236,16 @@ get_k8s_master_domain() {
     return 1
 }
 
+check_kops_version() {
+  which kops
+  KOPS_VERSION="$( kops version | awk '{print $2}' )"
+  echo "${KOPS_VERSION#1.} >= 7.1" | bc
```

Comment:
Seems like this is a debug statement left in here?

Reply:
This is actually the version check :-). Basically, I needed a way to tell if e.g. "1.7.1" was greater than "1.6.9" in bash, so I wrote hilarious, sad hack in which I strip the leading "1." and then echo an expression to "bc". So this becomes `echo "7.1 > 6.9" | bc` which then prints "0" if it's false and "1" if it's true